### DISCUSSION TITLE
Zepp OS New Devices Released, Amazfit Active and Amazfit Active Edge

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#259] [Announcements]
![Jietu20231027-173028](https://github.com/zepp-health/zeppos-forum/assets/107526244/730bb18d-502e-481c-84c7-7bac2c87b19d)

https://www.amazfit.com/products/amazfit-active

![Jietu20231027-173046](https://github.com/zepp-health/zeppos-forum/assets/107526244/e26bee8c-bd2d-48f1-a9ea-4cb8e4dfb959)

https://www.amazfit.com/products/amazfit-active-edge

Meanwhile, our Simulator is ready for developers to create cool programs for the new device!

**Comment by SashaCX75** [comment]
On the page. 
https://docs.zepp.com/docs/reference/related-resources/device-list/
There are two identical deviceSources for Amazfit Active. I think one of them needs to be changed.
